src/app/reservation/[roomId]/page.tsx

```diff
@@ -0,0 +1,18 @@
+import Head from 'next/head'
+
+type RoomReservationParams = {
+    roomId: number
+}
+
+export default function RoomReservation({ params }: { params: RoomReservationParams }) {
+    return (
+        <div>
+            <Head>
```

Comment:
In Next 13 and above with app router we don't use <Head> tag for attaching metatags like `title` instead you can use an object like shown here: https://nextjs.org/docs/app/building-your-application/optimizing/metadata